src/config/offlineDB.connection.js

```diff
@@ -3,8 +3,8 @@ import Dexie from 'dexie'
 const DB_DEFAULT_NAME = 'tknbrgoff'
 
 const dbInstance = new Dexie(process.env.VUE_APP_OFFLINE_DB || DB_DEFAULT_NAME)
-dbInstance.version(2).stores({
-  transactions: `transactionHash, *accountsAddresses, networkId, timestamp, blockNumber, receiveAmount, currentStep, type, senderAddress, receiverAddress`,
+dbInstance.version(4).stores({
```

Comment:
I'd maybe add some comment on the README explaining that each modification to the data schema requires this version to be bumped